src/schema/hotel-price-crawling.json

```diff
@@ -129,6 +129,46 @@
                 },
                 "canBeShared": {
                     "type": "boolean"
+                },
+                "bed": {
+                    "type": "array",
+                    "description": "Types of the beds.",
+                    "minItems": 0,
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "single",
+                            "double",
+                            "semi-double",
+                            "queen",
+                            "king",
+                            "twin",
```

Comment:
so, it may be something like `enum: [single, semi-double, double, queen, king]`, and twin is just single added twice:
```js
room: {
  rates: [...],
  ...,
  beds: [
    'single',
    'singe'
  ],
  ...
```

Reply:
3 beds - 3 elements in `beds` field, 4 beds - 4 elements, but how do we communicate 'more than 4' (not 5 or 6, but 5+)?

Reply:
Well, there is no place for '4 or more' in transaction XML, if it's 7 it just expects 7 `<Bed>` elements.  So, let's do the same in our data, just store 7 elements in array for 7 beds. 99 for 99 beds.